### ISSUE TITLE
VVSG unique ballot identifiers / scanning imprinting

### ISSUE THREAD

**Issue by mattroe** [votingworks/vxsuite-roadmap#31]
**What is the feature or problem to solve?**
VVSG requires that a paper-based voting system must be capable of adding a unique identifier after a voter casts their ballot. Although not all jurisdictions may use this feature, voting systems are required to have the capability to add a unique identifier to ballots. In practice, this requires scan imprinting functionality on our scanners.

**Why is it important?**
Required for EAC certification.

**Comment by mattroe** [comment]
Can we meet this requirement by only offering imprinting centrally? If a jurisdiction needs this, they can centrally scan all ballots and we already have a COTS hardware solution from Fujitsu.

**Comment by mattroe** [comment]
Discussed here: https://votingworks.slack.com/archives/CE7LM9LQ2/p1647637323664739

**Comment by adghayes** [comment]
I have assumed (as @benadida may have done a bit of research on as well) that the easiest past to imprinting, from a development and usability perspective, would be to use the imprinters available on the Fujitsu scanners if we can make those work. Please correct me if I am wrong.

I would like to call out that means that we will have to test imprinters for every Fujitsu scanner we sell, unless we can meet the VVSG requirement with one scanner and that is enough.

I'm also assuming we'll first want to do this with the fi-8170, since the fi-7160 is being EOL'd. The fi-8170 has a different imprinter than the fi-7160, part number: https://fujitsuscannerstore.com/pa03810-d201/. 

So as a pre-requisite to this work, we'll want to prioritize votingworks/vxsuite#1741. I think that might be a good task for me, once I get to this project, to begin familiarizing myself with the Fujitsu APIs. @mattroe I plan to order another fi-8170 to myself for that work.

**Comment by mattroe** [comment]
@adghayes yes, I've been operating under the assumption we would utilize the Fujitsu imprinter attachments, and we should definitely include supporting the latest Fujitsu scanners as part of this work.

I'll also add a more complete doc of context for this problem before you end up picking it up.
